microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -62,6 +62,10 @@ class DpdkTestpmd(Tool):
         r"dpdk-(?P<major>[0-9]+)\.(?P<minor>[0-9]+)"
     )
 
+    _debian_get_arch_from_kernel_version_regex = re.compile(
```

Comment:
Add examples above. And there is hardware_platform, which is arch, in kernel_information already.

Reply:
I think I'm actually looking for the 'prerelease' or 'build' field in the semver. It's used for the package naming scheme for linux-image-[build version]. I'll poke around and see if it's picked up by the kernel tool.